packages/editor/src/utils/CmsTooltip.test.ts

```diff
@@ -0,0 +1,53 @@
+import { describe } from 'vitest';
+import type { ContentObject } from '@axonivy/form-editor-protocol';
+import { findCmsEntries } from './CmsTooltip';
+
+describe('test tooltips', () => {
```

Comment:
If I only have one test I usually don't use a describe, as it is for me a bit a cluster tests together thing...